sql/sqlite/sqlspec.go

```diff
@@ -114,14 +136,26 @@ func schemaSpec(s *schema.Schema) (*specutil.SchemaSpec, error) {
 
 // tableSpec converts from a concrete SQLite sqlspec.Table to a schema.Table.
 func tableSpec(tab *schema.Table) (*sqlspec.Table, error) {
-	return specutil.FromTable(
+	spec, err := specutil.FromTable(
 		tab,
 		columnSpec,
 		specutil.FromPrimaryKey,
 		indexSpec,
 		specutil.FromForeignKey,
 		specutil.FromCheck,
 	)
+	if err != nil {
+		return nil, err
+	}
+	for _, a := range tab.Attrs {
+		switch a.(type) {
+		case *WithoutRowID:
+			spec.Extra.SetAttr(schemahcl.BoolAttr("without_rowid", true))
+		case *Strict:
+			spec.Extra.SetAttr(schemahcl.BoolAttr("strict", true))
+		}
+	}
```

Comment:
Embed it like [this](https://github.com/ariga/atlas/blob/master/sql/internal/specutil/convert.go#L394-L400) to ensure it's added **after** the columns.